packages/headless/src/features/facets/facet-search-set/facet-search-reducer-helpers.ts

```diff
@@ -15,6 +15,10 @@ export type FacetSearchState<T extends FacetSearchResponse> = {
    * The facet search response.
    */
   response: T;
+  /** The initial maximum number of values to fetch.
+   * @defaultValue `10`
```

Comment:
I think we can leave out the default value here because it's not something that can be configured, and it depends on the `numberOfValues`. I see it similar to `isLoading` not having a default value of `false`, `response` having a default value of the empty response object etc.